ethereum/vyper
# Update the hex of `rlp_decoder_tx` in comment

### - What I did
The `rlp_decoder_tx` changed and the tx hex and contract address in the instruction in comment is therefore stale. So I just update them.

### - How I did it

Karl told me about this modification. The hex also used in `./tests/setup_transaction_tests.py`. Besides, I verified the address of the tx created from the hex.

### - How to verify it

Create a tx with the hex using `rlp.decode` and check if its address is equaled to the address `RLP_DECODER_ADDRESS`

### - Cute Animal Picture
![hqdefault](https://user-images.githubusercontent.com/8223657/33269542-0b9e6acc-d3bc-11e7-8145-f81c27b0c0d0.jpg)
